docs/extensions/referenced-resources.md

```diff
@@ -26,7 +26,7 @@ spec:
       name: my-foobar-secret
 ```
 
-Gardener expects to find these referenced resources in the project namespace (e.g. `garden-dev`) and will copy them to the Shoot namespace in the Seed cluster when reconciling a Shoot, adding a prefix to their names to avoid naming collisions with Gardener's own resources. 
```

Comment:
Revert the file name change.
Revert it for `docs/extensions/controller-registration.md` as well.

Reply:
I agree for reverting the change to `controller-registration.md`, as the resource name is `ControllerRegistration`. 

However, I still think that `Referenced Resources` topic name should be split for readability, as it is essentially two separate words.

Reply:
I updated the comment, it is rather for reverting the name for `docs/extensions/managed-resources.md` as I am not allowed to comment in the affected file.